configure.ac

```diff
@@ -863,6 +863,10 @@ export device_name
 export device_args
 export devicedir
 
+if test "$device_name" = "ch4" ; then
+    AC_DEFINE([ENABLE_LOCAL_SESSION_INIT], 1, [Define to skip initializing builtin world comm during MPI_Session_init])
+fi
+
```

Comment:
I'm not sure why we do anything but abort if the user calls session init with ch3. Is there intent to fully support non-world init in ch3?

Reply:
Most sessions applications will work fine with `ch3` given that every process will call `MPI_Session_init`. So I don't see why we need force the abort.

Reply:
OK, fair enough. I don't think its actively harming anything. Unless we end up with a bunch of support request from confused users it is fine to leave in.